tests/integration/pure_cairo_vm/cheatcodes/invoke/invoke_test.cairo

```diff
@@ -0,0 +1,50 @@
+from starkware.cairo.common.math import assert_not_zero
+
+func test_invoke_without_transformation(){
```

Comment:
Please test it through proxy contract

Reply:
We should do the same for the call cheatcode

@karol-bisztyga can you please add such tests?

Reply:
#1361